spec/junk_drawer/rails/bulk_updatable_spec.rb

```diff
@@ -131,4 +77,8 @@
     expect(models.last.boolean_value).to be false
     expect(models.last.string_value).to eq 'plop'
   end
+
+  DATA_TYPES.each do |type|
+    it_behaves_like 'bulk updatable type', type
```

Comment:
Maybe this is just ruby/rspec magic, but `'bulk updatable type'` just calls `spec/support/shared_examples/bulk_updatable_type.rb`?

Reply:
It looks for shared examples with that exact description, so the line below `RSpec.shared_examples 'bulk updatable type'` declares it, and this makes use of it.